### ISSUE TITLE
How about a release?

### ISSUE THREAD

**Issue by bramski** [Ibsciss/ruby-middleware#5]
This gem looks alright.  How about releasing something?

**Comment by lilobase** [comment]
It is already released on rubygems : https://rubygems.org/gems/ibsciss-middleware/ 
Or do you have something else in mind ?

**Comment by bramski** [comment]
![screen shot 2016-09-27 at 8 53 14 am](https://cloud.githubusercontent.com/assets/421781/18881370/df8d3bc4-848f-11e6-8d41-a06379f8e618.png)

**Comment by bramski** [comment]
It's quite tricky to look at your project and figure out where the releases were and what was fixed.  I was trying to get a gauge yesterday of the maturity of the project and went diving through the code and rubygems to figure out where it was at.  Most github projects are keeping these relatively in sync by having some releases with some release notes about what's been put out.

The code of the project does look of good quality though; so I chose to use it for our project.  But I'd hope to save work for future users.
